src/PoolManager.sol

```diff
@@ -51,6 +51,7 @@ contract PoolManager is Auth, IPoolManager {
     TrancheTokenFactoryLike public trancheTokenFactory;
     ERC7540VaultFactory public vaultFactory;
     RestrictionManagerFactoryLike public restrictionManagerFactory;
+    address public cfgRouter;
```

Comment:
nit: I'm not a fan of using the cfg abbreviation since it implies the CFG token. I'd just call this router

```suggestion
    address public router;
```